upup/models/cloudup/resources/addons/authentication.aws/k8s-1.12.yaml.template

```diff
@@ -19,10 +103,8 @@ spec:
       labels:
```

Comment:
As I've noted in #8200, the `scheduler.alpha.kubernetes.io/critical-pod` annotation is no longer supported as of k8s 1.16.